addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -54,69 +54,50 @@
             <field name="name">mailing.mailing.form</field>
             <field name="model">mailing.mailing</field>
             <field name="arch" type="xml">
-                <form string="Mailing" class="o_mass_mailing_mailing_form">
-                    <header style="min-height:31px;">
-                        <button name="action_launch" type="object" class="oe_highlight" string="Send"
-                            attrs="{'invisible': [('state', 'in', ('in_queue',  'sending', 'done'))]}"
-                            confirm="This will send the email to all recipients. Do you still want to proceed ?"/>
-                        <button name="action_schedule" type="object" class="btn-secondary" string="Schedule"
-                            attrs="{'invisible': [('state', 'in', ('in_queue',  'sending', 'done'))]}"/>
-                        <button name="action_test" type="object" class="btn-secondary" string="Test"/>
-                        <button name="action_cancel" type="object" attrs="{'invisible': [('state', '!=', 'in_queue')]}" class="btn-secondary" string="Cancel"/>
-                        <button name="action_retry_failed" type="object" attrs="{'invisible': ['|', ('state', '!=', 'done'), ('failed', '=', 0)]}" class="oe_highlight" string="Retry"/>
+                <form string="Mailing" class="o_mass_mailing_mailing_form" js_class="mass_mailing_form">
+                    <div class="o_mass_mailing_sheet">
+                        <header style="min-height:31px;">
```

Comment:
Form view should not hold inline styling. A minimal CSS is accepted in css files, especially here as it is quite custom but form view should be as light as possible to ease its edition.